client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -601,6 +609,43 @@
         };
     }
 
+    PackagingEmbeddedDirective.$inject = ['packages'];
+    function PackagingEmbeddedDirective(packages) {
+        return {
+            templateUrl: 'scripts/superdesk-packaging/views/packaging.html',
+            require: '^sdAuthoringContainer',
+            scope: {
+                item: '='
+            },
+            link: function(scope, elem, attrs, authoringCtrl) {
+                scope.$watch('item', function(item) {
+                    if (item && item.lockIt){
+                        scope.lock();
+                    } else {
+                        scope.origItem = null;
+                        scope.$applyAsync(function() {
```

Comment:
Isn't this call to `$applyAsync()` redundant? The code here is already wrapped in a `$watch`, meaning that it is executed during `$digest`, which runs at least twice anyway to check whether or not any of the `$watch` handlers made a yet another change to the model.

Or am I missing something? It's not really clear to me why we first set `origItem` to `null`, and soon after that to `item'. Thanks for any explanation in advance!